vignettes/rhino-fluent.Rmd

```diff
@@ -0,0 +1,321 @@
+---
+title: "Tutorial: Build a Rhino application using shiny.fluent"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{Tutorial: Build a Rhino application using shiny.fluent}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: sentence
+---
+
+This quick tutorial helps to build a basic RShiny application with the structure provided by the `Rhino` package and simultaneously using the `Shiny.fluent` package.
+It introduces the user to the concepts of -
+
+-   Rhino project structure for a RShiny application and get it running
```

Comment:
This is not the goal of this tutorial. This tutorial is about building a Rhino application using shiny.fluent. The tutorial about building a Shiny application using Rhino already exists in Rhino documentation.